examples/read_parquet_cloud/src/main.rs

```diff
@@ -1,6 +1,7 @@
-use std::env;
+use std::collections::HashMap;
```

Comment:
Can you use `PlHashMap`? That's the hashmap we use within polars.

Reply:
@ritchie46 sorry about this the interface to `object_store` uses a hash map. Let me see if I can use a different API on their side, if not I can use a PLHashMap and translate it just before the call.

Reply:
oh, in that case nevermind.

Reply:
Ugh, I was totally wrong and you were right to push on this. The initial version of the API did use a HashMap but in the review process they removed it. I adapted the improved API and also removed the HashMap.